src/main/kotlin/org/taktik/icure/entities/embed/TimeTableItem.kt

```diff
@@ -27,11 +27,12 @@ import com.github.pozo.KotlinBuilder
 @JsonIgnoreProperties(ignoreUnknown = true)
 @KotlinBuilder
 data class TimeTableItem(
-	val days: List<String> = emptyList(),
+	val rrule: String? = null,
```

Comment:
Could you please add some KDoc to explain what it the rrule purpose and which format it needs to follow to replace days & recurrenceTypes properly ? 
Precise it follows the standard RFC-5545